runtime/libia2/include/ia2.h

```diff
@@ -174,6 +174,13 @@ uint32_t ia2_get_pkru();
 /// Returns the current compartment pkey
 size_t ia2_get_pkey();
 
+/// lib must be a single DSO or "main" for the executable
```

Comment:
Using `NULL` for the executable might be more efficient than a hard-coded string.